pkg/apis/serving/v1alpha3/framework_custom.go

```diff
@@ -0,0 +1,21 @@
+package v1alpha3
+
+import v1 "k8s.io/api/core/v1"
+
+// CustomFramework defines arguments for configuring ONNX model serving.
+type CustomFramework struct {
+	v1.PodSpec `json:",inline"`
```

Comment:
Can we consider using knative's [revision template spec](https://github.com/knative/serving/blob/master/pkg/apis/serving/v1/revision_types.go#L63)? we have complains in the past that they can not set annotations at component level as well as fields like `ContainerConcurrency` and `TimeoutSeconds`.

Reply:
I'm not sure I'm ready to so tightly couple ourselves to Knative interfaces. We've had requests for not knative implementations and I expect we may be seeing that soon.

Reply:
Being able to set `ContainerConcurrency` and `TimeoutSeconds` is a must-have imo.

Reply:
We currently expose "Parallelism" in the Extensions spec. Maybe we can expose TimeoutSeconds there as well? WDYT?

Reply:
https://github.com/kubeflow/kfserving/pull/806 re. `ContainerConcurrency`. I think it makes a lot of sense to expose `TimeoutSeconds` in the spec. I believe a timeout will be relevant even for other backends than KNative.